src/main.cpp

```diff
@@ -610,7 +610,7 @@ MiningCPID GetNextProject(bool bForce)
 
 
 
-    if (GlobalCPUMiningCPID.projectname.length() > 3   &&  GlobalCPUMiningCPID.projectname != "INVESTOR"  && GlobalCPUMiningCPID.Magnitude > 1)
+    if (GlobalCPUMiningCPID.projectname.length() > 3   &&  GlobalCPUMiningCPID.projectname != "INVESTOR"  && GlobalCPUMiningCPID.Magnitude >= 1)
```

Comment:
Could this be changed to greater than 0?

Reply:
i dont think so i think it is too prevent staking a por with less then 1 magnitude. as if u get lower then 1 mag its a magnitude too low for por and i would also have to check to make sure that if this did populate the magnitude fields that it wouldnt affect a pos interest block due to the magnitude rules.